src/Core/src/Platform/iOS/LabelExtensions.cs

```diff
@@ -23,5 +23,16 @@ public static void UpdateTextColor(this UILabel nativeLabel, ILabel label)
 				nativeLabel.TextColor = textColor.ToNative(textColor);
 			}
 		}
+
+		public static void UpdateCharacterSpacing(this UILabel nativeLabel, ILabel label)
+		{
+			if (string.IsNullOrEmpty(label.Text))
+				return;
+
+			var textAttr = nativeLabel.AttributedText?.AddCharacterSpacing(label.Text, label.CharacterSpacing);
```

Comment:
This logic with the attributed text is very different to the decorations PR. That one check for things, this just uses text. Will this override decorations?